TeslaLib/ITeslaClient.cs

```diff
@@ -22,7 +22,7 @@ public interface ITeslaClient
         List<TeslaVehicle> LoadVehicles();
         Task<List<TeslaVehicle>> LoadVehiclesAsync(CancellationToken cancellationToken);
         Task LoginAsync(string password, string mfaCode = null);
-        Task LoginUsingTokenStoreAsync(LoginToken loginToken, bool forceRefreshOlderThanToday = false);
+        Task RefreshTokenFromStoreAsync( bool forceRefresh = false);
```

Comment:
Woah, what's going on here?  This LoginUsingTokenStore method is the best way we have to log in.  The caller loads the token from the store, and we log in here, updating any necessary internal data structures.  

Perhaps we could move something around - there may be a better approach.  But we can't log in without this method.

Reply:
Remove the newly added method RefreshTokenFromStoreAsync